### PR TITLE
Allow to add links to Youtube or Soundcloud media as attachment (fixes #...

### PR DESCRIPTION
**Pull request by gutard**
...1340)

### CODE REVIEW

**Comment by camillemonchicourt** [comment]
Only on treks or also on POIs ?

**Comment by gutard** [comment]
> Only on treks or also on POIs ?

All objects with attachments.

**Comment by camillemonchicourt** [comment]
OK thanks

File: conf/buildout.cfg [hunk 1]
```diff
@@ -111,6 +111,7 @@ easy-thumbnails = 1.4
 simplekml = 1.2.1
 djangorestframework = 2.4.2
 djangorestframework-gis = 0.7
+django-embed-video = 0.11
```
Comment (leplatrem) [inline conf/buildout.cfg]: The django eco system is amazing :)

File: docs/changelog.rst [hunk 2]
```diff
@@ -7,6 +7,8 @@ CHANGELOG
 
 **Breaking changes**
 
+* Before running install, run this SQL command to add a column for video attachments:
+  `ALTER TABLE fl_t_fichier ADD COLUMN attachment_video VARCHAR(200) NOT NULL DEFAULT '';`
```
Comment (leplatrem) [inline docs/changelog.rst]: Double `` ` ``

File: docs/changelog.rst [hunk 3]
```diff
@@ -14,6 +16,7 @@ CHANGELOG
 **New features**
 
 * Publication workflow (fixes #1018)
+* Allow to add links to Youtube or Soundcloud media as attachment
```
Comment (leplatrem) [inline docs/changelog.rst]: Vimeo too :)

Comment (gutard) [inline docs/changelog.rst]: I'm not sure we will provide integration code for both Youtube And Vimeo in Geotrek-Rando

Comment (leplatrem) [inline docs/changelog.rst]: It's just that I saw it among available backends in django-embed-video :blush:

**Pull request merged**